db/migrate/20221019174254_add_user_ref_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :author, foreign_key: { to_table: :users }
+  end
+end
```

Comment:
- [ ] I was unable to set up your app locally because this migration already exists. Kindly remove this file to fix that.